gfx/drivers/d3d11.c

```diff
@@ -1025,13 +1025,9 @@ d3d11_gfx_init(const video_info_t* video, const input_driver_t** input, void** i
 
    font_driver_init_osd(d3d11, false, video->is_threaded, FONT_DRIVER_RENDER_D3D11_API);
 
-   if (settings->bools.video_shader_enable)
-   {
-      const char* ext = path_get_extension(retroarch_get_shader_preset());
-
-      if (ext && !strncmp(ext, "slang", 5))
-         d3d11_gfx_set_shader(d3d11, RARCH_SHADER_SLANG, retroarch_get_shader_preset());
-   }
+   const char *shader_preset   = retroarch_get_shader_preset();
```

Comment:
Same C89 issue here. Variables need to be declared either at the top of a code block or function.